src/CloudNative.CloudEvents.Protobuf/CloudNative.CloudEvents.Protobuf.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+	<Description>Support for the Protobuf event format in for CloudNative.CloudEvents</Description>
+	<PackageTags>cncf;cloudnative;cloudevents;events;protobuf</PackageTags>
+	<LangVersion>8.0</LangVersion>
```

Comment:
can we use 9?

Reply:
We can (or even 10), but we've got 8.0 everywhere at the moment, so I'll stay consistent with that. We can change it in all projects later.